src/components/IdentityLine.tsx

```diff
@@ -20,14 +24,22 @@ export const IdentityLine: React.FC<IdentityLineProps> = ({ identity }) => {
         <Flex>
           <Value>{value}</Value>
         </Flex>
-        {primary && status === "validated" && <p>Primary</p>}
+        {primary && status === "validated" && (
+          <PrimaryBadge>
+            <p>Primary</p>
+            <PrimaryIcon />
+          </PrimaryBadge>
```

Comment:
This should be inside the `PrimaryBadge` (i.e. here, it should be `<PrimaryBadge/>`).

Reply:
OK fixed 6ed6a28